assets/queries/openAPI/2.0/non_oauth2_security_requirement_defining_oauth2_scopes/test/positive2.yaml

```diff
@@ -0,0 +1,26 @@
+swagger: "2.0"
+info:
+  title: Simple API Overview
+  version: 1.0.0
+paths:
+  "/":
+    get:
+      operationId: listVersionsv2
+      summary: List API versions
+      responses:
+        "200":
+          description: 200 response
+      parameters:
+        - name: limit2
+          in: body
+          description: max records to return
+          required: true
+          schema:
+            type: object
+security:
```

Comment:
Equal to the previous [PR](https://github.com/Checkmarx/kics/pull/3701#discussion_r655392318)